frontend/src/data/utils/datasetutils.ts

```diff
@@ -142,6 +142,17 @@ export function percent(numerator: number, denominator: number): number | null {
     : Math.round((1000 * numerator) / denominator) / 10;
 }
 
+/** Finds expected value of an ailment based on a population sample. */
+export function estimateTotal(
+  numerator: number,
```

Comment:
Might be clearer to rename the arguments to this function to sample_count, sample_population, and total_population

Reply:
For sure